l10n_fi_payment_reference/models/account_invoice.py

```diff
@@ -115,7 +115,10 @@ def _compute_payment_reference(self):
     def invoice_validate(self):
         res = super(AccountInvoiceFinnish, self).invoice_validate()
         # noinspection PyProtectedMember
-        self.filtered(lambda i: i.payment_reference_type != 'none' and
-                      i.type == 'out_invoice') \
-            ._compute_payment_reference()
+        self.filtered(
+            lambda i:
+                i.payment_reference_type != 'none'
```

Comment:
Please add parentheses. This kind of combination of ANDs and ORs without them is confusing.

Reply:
Fixed.